lib/pivotal-tracker/account_membership.rb

```diff
@@ -0,0 +1,27 @@
+class PivotalTracker::AccountMembership
+  attr_accessor :kind, :id, :person, :account_id, 
+                :created_at, :updated_at, :owner, :admin,
+                :project_creator, :timekeeper, :time_enterer
+
+  def initialize(membership_attributes)
+    membership_attributes.each do |attribute, value|
+      self.send(:"#{attribute}=", value)
+    end
+  end
+
+  def self.all(account_id)
+    response = PivotalTracker.get("/accounts/#{account_id}/memberships")
+    if response.code == 200
+      response.parsed_response.map do |membership|
+        PivotalTracker::AccountMembership.new(membership)
+      end
+    else
+      puts response.inspect
```

Comment:
@wlffann if we merge #5 first, this error should grab the constant that will be in the PivotalTracker model.